doc/nrf/working_with_nrf/nrf53/nrf5340_gs.rst

```diff
@@ -0,0 +1,109 @@
+.. _ug_nrf5340_gs:
+
+Getting started with nRF5340 DK
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section gets you started with your nRF5340 :term:`Development Kit (DK)` using the |NCS|.
+It tells you how to install the :ref:`peripheral_uart` sample and perform a quick test of your DK.
+
+If you have already set up your nRF5340 DK and want to learn more, see the following documentation:
+
+* :ref:`ug_nrf5340` for more advanced topics related to the nRF5340 DK if you are already familiar with the |NCS|.
+* The :ref:`introductory documentation <getting_started>` for more information on the |NCS| and the development environment.
+
+.. _nrf5340_gs_requirements:
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF5340 DK
+* micro-USB 2.0 cable
+
+Software
+========
+
+On your computer, one of the following operating systems:
```

Comment:
I think we have already covered this in [this sentence](https://github.com/nrfconnect/sdk-nrf/pull/10234/files#diff-4306881ff8c514a522d0559898809372bcea37ad8afee76ea7db991a98f9b2b8R23).
```suggestion
```

Reply:
I kept the "On your computer:" to contrast with "On your mobile device:"